python/palindrome_checker.py

```diff
@@ -0,0 +1,14 @@
+'''
+
+                            Palindrome checker by python
+
+'''
+def palindrome (s):
+    r=s[::-1]
```

Comment:
![image](https://user-images.githubusercontent.com/44049528/94264908-38e42400-ff55-11ea-8114-e3d3246b9ea2.png)
You remove these codes and make a issue asking to write a code for it.